---a/net/minecraft/world/gen/layer/GenLayerFuzzyZoom.java
+++b/net/minecraft/world/gen/layer/GenLayerFuzzyZoom.java
@@ -1,17 +1,15 @@
 package net.minecraft.world.gen.layer;
 
-public class GenLayerFuzzyZoom extends GenLayerZoom
-{
+public class GenLayerFuzzyZoom extends GenLayerZoom {
+
     private static final String __OBFID = "CL_00000556";
 
-    public GenLayerFuzzyZoom(long p_i2123_1_, GenLayer p_i2123_3_)
-    {
+    public GenLayerFuzzyZoom(long p_i2123_1_, GenLayer p_i2123_3_) {
         super(p_i2123_1_, p_i2123_3_);
     }
 
-    protected int selectModeOrRandom(int p_151617_1_, int p_151617_2_, int p_151617_3_, int p_151617_4_)
-    {
-        return this.selectRandom(new int[] {p_151617_1_, p_151617_2_, p_151617_3_, p_151617_4_});
+    protected int selectModeOrRandom(int p_151617_1_, int p_151617_2_, int p_151617_3_, int p_151617_4_) {
+        return this.selectRandom(new int[]{p_151617_1_, p_151617_2_, p_151617_3_, p_151617_4_});
     }
-}
+}
